src/app/new-auth/email-verified/email-verified.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, OnInit } from '@angular/core';
```

Comment:
What is this email-verified component for?

Reply:
This component is the redirect when you click on the email that is sent to you to verify your email. This is not the same email sent in this PR.